### DISCUSSION TITLE
NordTronv2, a spin on the OG NordTron

### DISCUSSION THREAD

**Discussion by faris0x** [JanDeDobbeleer/oh-my-posh#4055] [Themes]
this theme was written using the original json of nordtron.omp.json, with some parts stolen from cern ;)
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/107783820/8eed1f8a-3bb3-430c-bf7b-83806936769d)

**Comment by faris0x** [comment]
[nordtronv2.omp.txt](https://github.com/JanDeDobbeleer/oh-my-posh/files/12030772/nordtronv2.omp.txt)
json of the theme in .txt because it wont let me upload .json >:(

**Reply by JanDeDobbeleer** [reply]
@WilliamAfton-codes you can add it as a code block.

**Comment by faris0x** [comment]
Eh, it’s good enough. Just rename the extension
